conf/cassandra.yaml

```diff
@@ -932,6 +932,13 @@ compaction_throughput_mb_per_sec: 64
 # between the sstables, reducing page cache churn and keeping hot rows hot
 sstable_preemptive_open_interval_in_mb: 50
 
+# Starting from 4.1 sstables support UUID based generation identifiers. They are disabled by default
+# because once enabled, there is no easy way to downgrade. When the node is restarted with this option
+# set to true, each newly created sstable will have UUID based generation identifier and such files are
+# not readable by previous Cassandra versions. At some point, this option will become true by default
+# and eventually get removed from the configuration.
+enable_uuid_generation_identifiers: false
```

Comment:
I'm wondering whether we should place this properties in the section for experimental features. @blerer wdyt?